hibernate-core/src/main/java/org/hibernate/metamodel/mapping/EntityMappingType.java

```diff
@@ -195,8 +196,7 @@ default EntityMappingType getRootEntityDescriptor() {
 		return superMappingType.getRootEntityDescriptor();
 	}
 
-	default TableReference locateTableReference(TableGroup tableGroup) {
-		return tableGroup.getPrimaryTableReference();
+	default void pruneForSubclasses(TableGroup tableGroup, Set<String> treatedEntityNames) {
```

Comment:
Can you please a javadoc to explain the use of this method?

Reply:
I'll add a comment